node/out_request.js

```diff
@@ -301,15 +301,6 @@ TChannelOutRequest.prototype.send = function send(arg1, arg2, arg3, callback) {
         self.span.name = endpoint;
     }
 
-    if (self.retryCount === 0) {
```

Comment:
this needs to be changed to `if (self.logical === false) {`

All direct requests ( host option ) and streaming requests go through this code path and not through `request.js` 

See how we emit latency stats for an example.